doc/man/8/rbd-nbd.rst

```diff
@@ -36,27 +36,26 @@ Options
 
 .. option:: --nbds_max *limit*
 
-   Override the parameter of NBD kernel module when modprobe, used to
-   limit the count of nbd device.
+   Override NBD kernel module parameters during modprobe. Used to
+   limit the maximum number of nbd devices.
 
 .. option:: --max_part *limit*
 
-    Override for module param nbds_max.
+    Override module parameter nbds_max.
 
 .. option:: --exclusive
 
-   Forbid writes by other clients.
+   Prevent other clients from writing to the image.
 
 .. option:: --io-timeout *seconds*
 
-   Override device timeout. Linux kernel will default to a 30 second request timeout.
-   Allow the user to optionally specify an alternate timeout.
+   Override device timeout. The Linux kernel defaults to a 30 second request timeout.
+   Allows the user to optionally specify an alternate timeout.
 
 .. option:: --reattach-timeout *seconds*
 
-   Specify timeout for the kernel to wait for a new rbd-nbd process is
-   attached after the old process is detached. The default is 30
-   second.
+   Specify timeout for the kernel to wait for a new rbd-nbd process to be
+   attached, after the old process is detached. The default is 30 seconds.
 
```

Comment:
This is a very good edit, and a substantial improvement to the text that was already present. Thank you for this work!

Reply:
In the service of making sure that this PR is properly formatted, I direct you to the portions of the following document that discuss the "Signed-off-by" line:

https://github.com/ceph/ceph/blob/master/doc/start/documenting-ceph.rst#commit-the-change

Here's the TL;DR, which is the actual text from the page at that link:

> Commits MUST include Signed-off-by: Firstname Lastname <email>. (strict)

I know that this may seem a bit discouraging, but the Jenkins infrastructure expects the "Signed-off-by" line to be present, or it will not permit the PR to be merged.